File: reto4/reto4.py
def createXmasTree(height, ornament):
    tree = ""

    for i in range(1, height + 1):
        padding = "_" * (height - i)
        tree += f"{padding}{ornament * (i + (i - 1))}{padding}\n"

    padding = "_" * (height - 1)
    tree += f"{padding}#{padding}\n"
    tree += f"{padding}#{padding}"

    return tree

File: reto4/reto4.ts
function createXmasTree(height: number, ornament: string): string {
  let tree = "";
  for (let i = 1; i <= height; i++) {
    const padding = "_".repeat(height - i);
    tree += `${padding + ornament.repeat(i + (i - 1)) + padding}\n`;
  }
  const padding = "_".repeat(height - 1);
  tree += `${padding}#${padding}\n`;
  tree += `${padding}#${padding}`;
  return tree;
}
